azurerm/helpers/azure/app_service.go

```diff
@@ -225,7 +225,6 @@ func SchemaAppServiceIdentity() *schema.Schema {
 					ValidateFunc: validation.StringInSlice([]string{
 						string(web.ManagedServiceIdentityTypeNone),
 						string(web.ManagedServiceIdentityTypeSystemAssigned),
-						string(web.ManagedServiceIdentityTypeSystemAssignedUserAssigned),
```

Comment:
we'll need to hard-code this value for the moment - I've got a PR tracking this here: https://github.com/Azure/azure-rest-api-specs/pull/8435